daemon/graphdriver/overlay2/overlay.go

```diff
@@ -179,6 +173,17 @@ func Init(home string, options []string, uidMaps, gidMaps []idtools.IDMap) (grap
 		return nil, err
 	}
 
+	if kernel.CompareKernelVersion(*v, kernel.VersionInfo{Kernel: 4, Major: 0, Minor: 0}) < 0 {
+		if !opts.overrideKernelCheck {
```

Comment:
Logic is reversed, should be `if opts.overrideKernelCheck`

Reply:
Oh! Yes, last-minute change before I pushed, thought to keep the Warning in place so made it an if/else; updating